Jenkinsfile

```diff
@@ -285,20 +285,30 @@ try {
               Utils.markStageSkippedForConditional("memcheckReleaseTest")
             }
           }
-        }, tpchQueryPlansAndVerification: {
-          stage("tpchQueryPlansAndVerification") {
+        }, tpchVerification: {
```

Comment:
(random line)
Just a thought: Should we split up `TableScan` into `TableScan (simple)` and `TableScan (complex)` to make it clearer that those expensive scans are actually ExpressionEvaluator invocations?

Reply:
You are talking about JOB?
It’s certainly an interesting information, but inner vs semi join is also super interesting or simple aggregates vs complex aggregates (multiple group columns or whatever). So I don’t know if we should really split this rather clear separation into operators and where that would lead to.

Reply:
Not only JOB.

The difference between TableScan and the named examples is that in your cases, those are just different variations of the same operator. In the case of the TableScan with the ExpressionEvaluator backend, we are effectively executing an entirely different operator. Remember that an EE scan could internally execute entire query plans. In the past, we have seen a lot of confusion with people wondering why the scan was so expensive.

Reply:
Hmm, I see your point but I am still not sure where this would end up (personally, I'd be more interested in the joins and the PQPs clearly state the scan implementation). I wouldn't veto though.

Reply:
No strong opinion about this - just a suggestion.

Reply:
Open a ticket about it and gather some opinions? I should not be more than the aggregation within the PQP visualization, right?

Reply:
Shouldn't be more than two lines or so, correct.

Your call how we want to proceed. Maybe just suggest it in the next fortnightly?